__mocks__/@actions/github.js

```diff
@@ -42,132 +42,151 @@ const PRs = [
+    user: { login: 'LitoMore' },
   },
   {
     //  5: PR that modifies one icon's source
     title: '504',
     number: 504,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     //  6: PR that modifies one icon's color and source
     title: '505',
     number: 505,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     //  7: PR that modifies an SVG and modifies that icon's color
     title: '506',
     number: 506,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     //  8: PR that modifies an SVG and modifies that icon's source
     title: '507',
     number: 507,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     //  9: PR that modifies an SVG and modifies that icon's color and source
     title: '508',
     number: 508,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     // 10: PR that adds an SVG and modifies another icon's color and source
     title: '509',
     number: 509,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     // 11: PR that adds an SVG and modifies another SVG
     title: '510',
     number: 510,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     // 12: PR that changes the data file but no metadata
     title: '511',
     number: 511,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     // 13: PR that adds a brand with a lowercased name
     title: '512',
     number: 512,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     // 14: PR that adds ACM (which should be listed before the lowercased brand)
     title: '513',
     number: 513,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     // 15: PR that adds a brand with an accented name
     title: '514',
     number: 514,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     // 16: PR that removes an icon
     title: '515',
     number: 515,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'develop' },
+    user: { login: 'LitoMore' },
   },
   {
     // Skipped release
     title: '[skip] Skip this one',
     number: 516,
     merged_at: '2011-01-26T19:01:12Z',
     base: { ref: 'master' },
+    user: { login: 'LitoMore' },
   },
   {
     // Previous release
     title: 'Release',
     number: 499,
     merged_at: '2011-01-02T19:01:12Z',
     base: { ref: 'master' },
+    user: { login: 'LitoMore' },
```

Comment:
Added a variety of maintainer names to the mock data.
To be applied along with the update to tests, also listed.

```suggestion
    number: 500,
    merged_at: null,
    base: { ref: 'develop' },
    user: { login: 'adamrusted' },
  },
  {
    //  1: PR that adds a file
    title: '501',
    number: 501,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'LitoMore' },
  },
  {
    //  2: PR that modifies an SVG
    title: '502',
    number: 502,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'mondeja' },
  },
  {
    //  3: PR that was merged before the previous release
    title: '498',
    number: 498,
    merged_at: '2011-01-01T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'PeterShaggyNoble' },
  },
  {
    //  4: PR that modifies one icon's color
    title: '503',
    number: 503,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'service-paradis' },
  },
  {
    //  5: PR that modifies one icon's source
    title: '504',
    number: 504,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'fbernhart' },
  },
  {
    //  6: PR that modifies one icon's color and source
    title: '505',
    number: 505,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'adamrusted' },
  },
  {
    //  7: PR that modifies an SVG and modifies that icon's color
    title: '506',
    number: 506,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'LitoMore' },
  },
  {
    //  8: PR that modifies an SVG and modifies that icon's source
    title: '507',
    number: 507,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'mondeja' },
  },
  {
    //  9: PR that modifies an SVG and modifies that icon's color and source
    title: '508',
    number: 508,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'PeterShaggyNoble' },
  },
  {
    // 10: PR that adds an SVG and modifies another icon's color and source
    title: '509',
    number: 509,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'service-paradis' },
  },
  {
    // 11: PR that adds an SVG and modifies another SVG
    title: '510',
    number: 510,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'fbernhart' },
  },
  {
    // 12: PR that changes the data file but no metadata
    title: '511',
    number: 511,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'adamrusted' },
  },
  {
    // 13: PR that adds a brand with a lowercased name
    title: '512',
    number: 512,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'LitoMore' },
  },
  {
    // 14: PR that adds ACM (which should be listed before the lowercased brand)
    title: '513',
    number: 513,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'mondeja' },
  },
  {
    // 15: PR that adds a brand with an accented name
    title: '514',
    number: 514,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'PeterShaggyNoble' },
  },
  {
    // 16: PR that removes an icon
    title: '515',
    number: 515,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'develop' },
    user: { login: 'service-paradis' },
  },
  {
    // Skipped release
    title: '[skip] Skip this one',
    number: 516,
    merged_at: '2011-01-26T19:01:12Z',
    base: { ref: 'master' },
    user: { login: 'fbernhart' },
  },
  {
    // Previous release
    title: 'Release',
    number: 499,
    merged_at: '2011-01-02T19:01:12Z',
    base: { ref: 'master' },
    user: { login: 'LitoMore' },
```